Mapper/Mapper.swift

```diff
@@ -342,6 +361,7 @@ public struct Mapper {
 
      - returns: The object for the given key
      */
+    @warn_unused_result
     private func JSONFromField(field: String) -> AnyObject? {
```

Comment:
My reasoning is that we have control on what happens `private`ly and this just seems overprotective. Is it realistic to think that `JSONFromField` will be misused?.